src/Nest/Aggregations/Bucket/BucketAggregate.cs

```diff
@@ -34,6 +34,20 @@ public class MultiBucketAggregate<TBucket> : IAggregate
 		public IReadOnlyCollection<TBucket> Buckets { get; set; } = EmptyReadOnly<TBucket>.Collection;
 	}
 
+	public class CompositeBucketAggregate : IAggregate
```

Comment:
if this were to derive from `MultiBucketAggregate<CompositeBucket>`, I think binary compatibility would be maintained.